lib/db.js

```diff
@@ -492,7 +494,7 @@ module.exports = (
     )
   }
 
-  DB.prototype.updateSessionToken = function (token, userAgentString) {
+  DB.prototype.updateSessionToken = function (token, userAgentString, ip) {
```

Comment:
we might need some validation for `userAgentString`, checking that it is exists, either in this function or where it is called.